### ISSUE TITLE
Various issues in Typst rendering with Quarto v1.4.547 (Mac)

### ISSUE THREAD

**Issue by ptram** [quarto-dev/quarto-cli#8366]
### Bug description

Hi,

Just a quick report of the issues I noticed when re-rendering an .md file after a few months. The previous PDF was made at the end of September 2023. No changes where made to the .md file. I only updated Pandoc and Quarto, and rendered it again.

- ToC title was "Contents", it is now "Table of contents".
- ToC is now cut after the first page.
- SVG images are not included, or just part of them are. I suspect inclusion of SVG files in Typst renderings are now following stricter rules, and an SVG-to-PDF filter has to be added somewhere in the source .md file.
- PNG images are not automatically resized, to fit the PDF column width. If the image is larger, it goes out of the right margin. I guess specifying a 100% width is now mandatory.

### Steps to reproduce

As described above, just rendering from Terminal after the update of Pandoc and Quarto. The command being:

`quarto render [source_filename]`

### Expected behavior

No particular behavior expected, but the same as in the previous rendering.

### Actual behavior

Differences as explained at the beginning.

### Your environment

BBEdit 14.6.9, VS Code 1.85.2
MacOS Monterey 12.7.2

### Quarto check output

```
Quarto 1.4.546
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.11: OK
      Dart Sass version 1.69.5: OK
      Deno version 1.37.2: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.4.546
      Path: /Applications/quarto/bin

[✓] Checking tools....................OK
      TinyTeX: (not installed)
      Chromium: (not installed)

[✓] Checking LaTeX....................OK
      Using: Installation From Path
      Path: /Library/TeX/texbin
      Version: 2022

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.11.4
      Path: /Library/Frameworks/Python.framework/Versions/3.11/bin/python3
      Jupyter: (None)

      Jupyter is not available in this Python installation.
      Install with python3 -m pip install jupyter

      There is an unactivated Python environment in .installbuilder. Did you forget to activate it?

[✓] Checking R installation...........(None)

      Unable to locate an installed version of R.
      Install R from https://cloud.r-project.org/
```

**Comment by mcanouil** [comment]
If you don't provide the means to reproduce as requested, it's very unlikely we'll be able to help or address possible bugs.

So, could you please share a small reproducible example?
Also, could you give the previous version of Quarto you used? (The date 2029 you give is clearly wrong and very unlikely to help us figure out the version)

**Comment by ptram** [comment]
> If you don't provide the means to reproduce as requested, it's very unlikely we'll be able to help or address possible bugs.

I'm not sure these are bugs. Maybe changes since the latest time I used Quarto (in September 2023 – sorry for the typo).

That is, the rendering of the same file achieve different results in the highlighted areas (SVG image inclusion, PNG image width).

The only possible bug being the cut ToC, that at the moment I don't know where to start investigating from. I've read the latest discussions and open issues, but I can't see anything specific to the issues I reported.

> So, could you please share a small reproducible example?

I'll try to extract something in the next days. The full source .md file is huge, and I can't even share it.

> Why are you updating Pandoc? Quarto comes with its own embedded version.

I keep it up-to-date for the other programs making use of it. But I ignored Quarto is self-sufficient even on this!
